### DISCUSSION TITLE
How to add MenuDivider to MenuDataItem.Children?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2839] [Q&A]
```cs
var menu = new MenuDataItem
{
    Name = "Options",
    Key  = "Options",

    Children = new[]
    {
        new MenuDataItem
        {
            Name = "Option 1",
            Key  = "Option 1",
        },
        new MenuDivider // Not allowed
        {
        },
        new MenuDataItem
        {
            Name = "Option 2",
            Key  = "Option 2",
        },
    },
};
```

Perhaps `Children` needs to be an array of a common interface or base class for `MenuDataItem` and `MenuDivider`?

**Comment by ElderJames** [comment]
Thanks for the suggestion @michaelcsikos , I see that antd has this implementation, so we will implement it as well. 
Would you be interested in helping?

![image](https://user-images.githubusercontent.com/7550366/197939326-a4f92d64-7684-439c-9af4-3dbfd7145e31.png)
